Sources/RInAppMessaging/Models/Requests/GetConfigRequest.swift

```diff
@@ -24,6 +26,7 @@ extension GetConfigRequest {
         queryItems.append(URLQueryItem(name: CodingKeys.appVersion.rawValue, value: appVersion))
         queryItems.append(URLQueryItem(name: CodingKeys.sdkVersion.rawValue, value: sdkVersion))
         queryItems.append(URLQueryItem(name: CodingKeys.locale.rawValue, value: locale))
+        queryItems.append(URLQueryItem(name: CodingKeys.locale.rawValue, value: rmcSDKVersion))
```

Comment:
This should be `CodingKeys.rmcSdkVersion.rawValue`?

Reply:
And we should check if `rmcSDKVersion` is not nil